README.md

```diff
@@ -513,10 +517,11 @@ SAAS
 <summary>Explain each of the following Cloud Computing Deployments:
 
   * Public
-  * Hybrid
```

Comment:
Hybrid was removed from the list

Reply:
Just moved the order Public, Hybrid, Private > Public, Private, Hybrid